action/builder.go

```diff
@@ -203,6 +204,20 @@ func (b *EnvelopeBuilder) BuildStakingAction(tx *types.Transaction) (Envelope, e
 	return b.build(), nil
 }
 
+// BuildRewardingAction loads rewarding action into envelope from abi-encoded data
+func (b *EnvelopeBuilder) BuildRewardingAction(tx *types.Transaction) (Envelope, error) {
```

Comment:
defined but not used?
need to use it in `web3server_utils.go` similar to how `BuildStakingAction` is used

Reply:
This function is prepared for another PR and that PR will use this function as similar `BuildStakingAction`, because accord the meeting discussed, this feature should split two PRs, one is this envelope actions and another is invoking API.

Reply:
ok i see

Reply:
cool, have added it.